web-ui/src/components/pulse/Pulse.test.jsx

```diff
@@ -4,15 +4,15 @@ import { fireEvent, render, screen } from '@testing-library/react';
 import Pulse from './Pulse';
 
 it('renders correctly', () => {
-  const component = renderer.create(
+  const component = render(
```

Comment:
I had just done some quite similar adjustments to this file. Standardizing to your version so no errors later on.